datashader/glyphs.py

```diff
@@ -32,25 +31,49 @@ def validate(self, in_dshape):
     @staticmethod
     @ngjit
     def _compute_x_bounds(xs):
+        if len(xs) == 0:
+            raise ValueError('x coordinate array is empty.')
```

Comment:
True; that would need an extra isnan(minval), etc. inside the loop. @jcrist points out that at this point the chunk of data has already been loaded, and so the len() should be cheap, so I'll leave it as-is.